src/components/landing/Splash.js

```diff
@@ -11,52 +14,90 @@ import Button from '@material-ui/core/Button';
+    height: '100vh',
   },
-  menuBar: {
-    background: '#011638',
+  toolBar: {
+    background: 'white',
+    color: '#434e61',
+    fontFamily: 'Nunito, sans-serif',
+    display: 'flex',
   },
-  title: {
-    color: '#ECECEC',
-    fontSize: '24px',
+  logo: {
+    height: '60px',
+    padding: '10px',
+  },
+  protein: {
+    marginRight: '9%',
+    height: '600px',
+    marginTop: '3%'
+  },
+  float: {
+    width: '100%',
+    textAlign: 'right',
   },
   button: {
-    color: '011638',
-    height: '200px',
-    width: '300px',
-    border: '#011638 1px solid',
-    // transition: '0.5s',
-    // backgroundSize: '200% auto',
-    // boxShadow: '0 0 20px #eee',
-    borderRadius: '1px',
-    // backgroundImage: 'linear-gradient(to right, #f83600 0%, #f9d423 51%, #f6d365 100%)',
+    marginRight: '20px'
+  },
+  createButton: {
+    // border: '3px solid #08AEEA',
+    // border: '3px solid #2AF598',
+    marginRight: '20px',
+  },
+  searchButton: {
+    background: 'linear-gradient(45deg, #08AEEA 30%, #2AF598 90%)',
+    borderRadius: 3,
+    border: 0,
+    color: 'white',
+    height: 48,
+    padding: '0 30px',
+    marginTop: '30px'
+  },
+  inlineFlex: {
+    display: 'inline-flex',
+    width: '100%',
+  },
+  titleContainer: {
+    marginLeft: '15%',
+    marginTop: '18%'
+  },
+  title: {
+    fontSize: '40px',
+  },
+  subTitle: {
+    fontSize: '20px',
+    fontWeight: '200',
+    width: '70%',
   },
-  buttonContainer: {
-    display: 'flex',
-    justifyContent: 'space-evenly',
-    marginTop: '100px',
-  }
 };
 
 function Splash(props) {
+  init();
   const { classes } = props;
 
   return (
     <div className={classes.root}>
       <AppBar position="static">
-        <Toolbar className={classes.menuBar}>
-          <Typography className={classes.title}>
-            Moltimate
-          </Typography>
+        <Toolbar className={classes.toolBar}>
+          <img className={classes.logo} src={Logo} />
+          <div className={classes.float}>
+            <Button className={classes.button}>About</Button>
+            <Button className={classes.button}>Publications</Button>
+            <Link to="/search">
+              <Button className={classes.button}>Search</Button>
+            </Link>
+            <Button className={classes.button}>Create</Button>
+          </div>
         </Toolbar>
       </AppBar>
-      <div className={classes.buttonContainer}>
-        <Link to="/search">
-          <Button className={classes.button}>Search</Button>
-        </Link>
-        <Link to="/create">
-          <Button className={classes.button}>Create</Button>
-        </Link>
+      <div className={classes.inlineFlex}>
+        <div className={classes.titleContainer}>
+          <Typography className={classes.title}>Moltimate Protein Analysis</Typography>
+          <Typography className={classes.subTitle}>Molecular visualization using catalytic site homology to predict unknown functions of proteins. </Typography>
```

Comment:
Small wording change, not sold on it either way. Did this line come from the sponsors or in ProMol? If so ignore me we can keep the same tag line as them.
```suggestion
          <Typography className={classes.subTitle}>Molecular visualization using catalytic site homology to predict proteins of unknown function.</Typography>
```

Reply:
Maybe "analyze proteins of unknown function"